pyproject.toml

```diff
@@ -7,7 +7,7 @@ ensure_newline_before_comments = true
 line_length = 88
 
 [tool.pylint.messages_control]
-disable = "bad-continuation,bad-whitespace,too-many-ancestors,too-few-public-methods,fixme"
+disable = "bad-continuation,bad-whitespace,too-many-ancestors,too-few-public-methods,fixme,fixme-info"
```

Comment:
`disable = "fixme-info"` is only needed when `pyproject.toml` is used.  It is not documented and beyond my comprehension.  
But now it works.